tools/proofers/character_selector.js

```diff
@@ -55,16 +55,28 @@ $(function () {
     const mruColumns = 12;
     const mruMax = 2 * mruColumns;
 
+    function setAlign(element, title) {
+        // to ensure the accents for Greek capital letters are visible, add a text indent.
+        // see also similar code in pinc/CharacterSelector.inc
+        if(title.startsWith("GREEK CAPITAL") && (title.includes("OXIA") || title.includes("VARIA"))) {
```

Comment:
I'm not a huge fan of duplicating logic in php and js. I wonder if we could have the logic once in php, and put a flag on the characters that require extra indent, and then keep that in the mru storage item?

Reply:
I'd rather do it by drawing all the pickers in javascript since we probably want to do that eventually in the new PI. But this might be encroaching on the new PI work.

Reply:
Agreed both that it would be nice to not duplicate this and also that I would defer working to abstract this into an API and consume it until we know more about where the new PI work is heading.